.bazelrc

```diff
@@ -7,3 +7,6 @@
 # The following workaround was suggested by https://github.com/grpc/grpc/issues/19645
 #
 build --copt -DGRPC_BAZEL_BUILD
+build --cxxopt=-std=c++17
+build --test_output=all
```

Comment:
I wouldn't add this here as it would be too noisy running locally, if you want this on CI add there.